### PR TITLE
aded SSH and Opaque types, renamed Custom to be more descriptive

### PR DESCRIPTION
**Pull request by dlaloue-vmware**
Added 2 missing standard secret types:  SSH and Opaque
Also renamed the auth type "Custom" to "AuthorizationHeader" to be more descriptive.

### CODE REVIEW

File: cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto [hunk 1]
```diff
@@ -222,6 +230,22 @@ message DockerCredentials {
   string email = 4;
 }
 
+// SshCredentials
+//
+message SshCredentials {
+  // private key
+  string private_key = 1;
+  // known hosts.
+  string known_hosts = 2;
```
Comment (absoludity) [inline cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto:239]: Should this be `repeated string known_hosts` to capture a list of known hosts? Fine either way, guessing you've talked about this in the meeting.

**Review by absoludity — APPROVED** [review]
LGTM. One question but fine either way.

File: cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto (hunk 1 above)
Comment (dlaloue-vmware) [reply cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto:239]: this is what i thought originally, but after reading about what known hosts means, and how it is used in carvel, it seems the expectation is more of it being the content of the known_hosts file that you would find in the .ssh/ folder

**Comment by antgamdia** [comment]
Great, thanks! I'll merge these changes into my UI branch.

**Pull request merged**